src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => century
+    ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+    : person.sex === 'm');
+
+  const sumOfAges = men.reduce((sum, man) =>
+    sum + (man.died - man.born), 0);
+
```

Comment:
Не розривай колбеки переносом, зменшується читабельність
```suggestion
  const sumOfAges = men.reduce(
    (sum, man) => sum + (man.died - man.born),
  ,0);

```